rpm/crash-reporter.spec

```diff
@@ -116,13 +116,6 @@ make %{_smp_mflags}
 /usr/lib/pkgconfig/*.pc
 
 %post
-# Remove timer unit symlink potentially left by old installation.
-rm -f /etc/systemd/system/basic.target.wants/crash-reporter-endurance.timer
```

Comment:
When this did exist?

Reply:
These removals came on 2014-04 and 2014-02. Should be safe by now.

Reply:
Ok, thanks for clarification.